Mathlib/Tactic/Linter/FlexibleLinter.lean

```diff
@@ -277,8 +281,12 @@ def stoppers : Std.HashSet Name :=
     ``cdot }
 
 /-- `SyntaxNodeKind`s that are allowed to follow a flexible tactic:
-  `simp`, `simp_all`, `simpa`, `dsimp`, `constructor`, `congr`, `done`, `rfl`, `omega`, `abel`,
-  `ring`, `linarith`, `nlinarith`, `norm_cast`, `aesop`, `tauto`, `fun_prop`, `split`, `split_ifs`.
+  `simp`, `simp_all`, `simpa`, `dsimp`, `grind`, `constructor`, `congr`, `done`, `rfl`, `omega`,
+  `abel` and `abel!`, `group`, `ring` and `ring!`, `module`, `norm_num`,
+  `linarith`, `nlinarith` and `nlinarith!`, `norm_cast`, `tauto`,
+  `aesop`, `cfc_tac` (and `cfc_zero_tac` and `cfc_cont_tac`),
+  `fun_prop`, `continuity` and `measurability`, `positivity`, `finiteness`, `finiteness`,
+  `split`, `split_ifs`.
```

Comment:
I wonder whether a better way to keep this in sync would be to write
```lean4
  { ``Lean.Parser.Tactic.simp, -- `simp`
  ...

```
below instead.

Reply:
So you propose removing the doc-string, and adding this to the list below? I like the fact that this has a doc-string... though I see that it never getting stale is also nice.